mlflow/store/tracking/rest_store.py

```diff
@@ -316,30 +313,62 @@ def start_trace(
         response_proto = self._call_endpoint(StartTrace, req_body)
         return TraceInfoV2.from_proto(response_proto.trace_info)
 
-    def start_trace_v3(self, trace: Trace) -> TraceInfo:
+    def start_trace_v3(self, trace_info: TraceInfo) -> TraceInfo:
         """
-        Start a trace using the V3 API format.
+        Create a new trace using the V3 API format.
 
-        NB: This method is named "Start" for internal reason in the backend, but actually
-        should be called at the end of the trace. We will migrate this to "CreateTrace"
-        API in the future to avoid confusion.
+        NB: The backend API is named "StartTraceV3" for some internal reason, but actually
+        it is supposed to be called at the end of the trace.
 
         Args:
-            trace: The Trace object to create.
+            trace_info: The TraceInfo object to create in the backend.
 
         Returns:
-            The returned TraceInfoV3 object from the backend.
+            The returned TraceInfo object from the backend.
         """
+        # NB: The Databricks backend expects a Trace object, not a TraceInfo object, although
+        # it doesn't use the data field at all. Trace data increases the payload size significantly,
+        # so we create a Trace object with an empty data field here.
+        trace = Trace(info=trace_info, data=TraceData(spans=[]))
         req_body = message_to_json(StartTraceV3(trace=trace.to_proto()))
-        response_proto = self._call_endpoint(
-            # NB: _call_endpoint doesn't handle versioning between v2 and v3 endpoint
-            # yet, so manually passing the v3 endpoint here.
-            StartTraceV3,
-            req_body,
-            endpoint="/api/3.0/mlflow/traces",
-            retry_timeout_seconds=MLFLOW_ASYNC_TRACE_LOGGING_RETRY_TIMEOUT.get(),
+
+        try:
+            response_proto = self._call_endpoint(
+                # NB: _call_endpoint doesn't handle versioning between v2 and v3 endpoint
+                # yet, so manually passing the v3 endpoint here.
+                StartTraceV3,
+                req_body,
+                endpoint=_V3_TRACE_REST_API_PATH_PREFIX,
+                retry_timeout_seconds=MLFLOW_ASYNC_TRACE_LOGGING_RETRY_TIMEOUT.get(),
+            )
+            return TraceInfo.from_proto(response_proto.trace.trace_info)
+        except Exception as e:
```

Comment:
```suggestion
        except MlflowException as e:
```

can we catch `MlflowException`?